transport/TransportImpl.cpp

```diff
@@ -2047,6 +2055,9 @@ void TransportImpl::onIceStateChanged(ice::IceSession* session, const ice::IceSe
                 _peerRtpPort = candidatePair.second.address;
                 _peerRtcpPort = candidatePair.second.address;
 
+                _transportType = endpoint->getTransportType();
+                _transportTypeIsSet = true;
```

Comment:
Shouldn't the _transportType be an Optional<> in the first place?

Reply:
Fixed.